engine/framework/model/master.go

```diff
@@ -52,7 +53,12 @@ type MasterMetaKVData struct {
 
 	// Config holds business-specific data
 	Config []byte `json:"config" gorm:"column:config;type:blob"`
-	// TODO: add master status and checkpoint data
+
+	// error message for the job
+	ErrorMsg string `json:"error-msg" gorm:"column:error_msg;type:text"`
```

Comment:
```suggestion
	ErrorMsg string `json:"error_message" gorm:"column:error_msg;type:text"`
```

for OpenAPI readability. And check other places

Reply:
I will unify all these name, to make it more readable and understandable.